internal/$L/$L.js

```diff
@@ -0,0 +1,100 @@
+/* global ILIB_MY_THEME_PATH */
+
+import {getIStringFromBundle} from '@enact/i18n/src/resBundle';
+import ResBundle from 'ilib/lib/ResBundle';
+
+// The ilib.ResBundle for the active locale used by $L
+let resBundle;
+
+/**
+ * Returns the current ilib.ResBundle
+ *
+ * @returns {ilib.ResBundle} Current ResBundle
+ */
+function getResBundle () {
+	return resBundle;
+}
+
+/**
+ * Creates a new ilib.ResBundle for string translation
+ *
+ * @param  {ilib.Locale} locale Locale for ResBundle
+ *
+ * @returns {Promise|ResBundle} Resolves with a new ilib.ResBundle
+ */
+function createResBundle (options) {
+	let opts = options;
+
+	if (typeof ILIB_AGATE_PATH !== 'undefined') {
+		opts = {
+			loadParams: {
+				// Deprecated; to be removed in future
+				root: ILIB_AGATE_PATH
```

Comment:
Since we don't have any legacy ilib support for agate, let's start off clean without this fallback.

Reply:
Updated